.github/pull_request_template.md

```diff
@@ -49,14 +49,4 @@ For pull requests that don't need to be mentioned at release time, use the `/rel
 ```release-note
 NONE
 ```
-
-Note, your release note has to meet the formatting restrictions noted above precisely, or else it will not be included.
-To validate its inclusion, run:
-
-```
-# export PR_NUM=<your PR number>
-wget -q -O- https://api.github.com/repos/shipwright-io/build/issues/${PR_NUM} | grep -oPz '(?s)(?<=```release-note..)(.+?)(?=```)' | grep -avP '\W*(Your release note here|action required: your release note here|NONE)\W*'
```

Comment:
@qu1queee can we try moving this to a test/job in our "Sanity checks" CI suite?

Doc here on how to use the embedded Github token to access the REST API: https://docs.github.com/en/actions/reference/authentication-in-a-workflow#example-2-calling-the-rest-api

Issue on how to extract the pull request ID in a Github action: https://github.com/actions/checkout/issues/58

Reply:
Good idea. The existing sanity checks apply also on scenarios before opening the PR, so I will not put it there. We will then need to introduce a different sanity check. I think I can do this in a different PR, I like it.

For this PR, the current proposed code `wget` does not work in macOS ( _only linux_ ), plus it fails today with this PR for example, while my `NONE` release note is not detected. I tried:

```sh
# export PR_NUM=740
# wget -q -O- https://api.github.com/repos/shipwright-io/build/issues/${PR_NUM} | grep -oPz '(?s)(?<=```release-note..)(.+?)(?=```)' | grep -avP '\W*(Your release note here|action required: your release note here|NONE)\W*'

```

I can fix the code, but as I said I dont see benefit on having this here, it just generates more manual steps when opening the PR. I can agree on removing this today, but opening a CI check for the notes validation ( _your idea Adam_ ). I think this will be a much better approach :) . Opinions?

Reply:
@gabemontero I believe this block was added because we had some release notes fail to generate?

Reply:
Yeah, this was added as a means to document for folks a way to verify whether their release notes were properly structured.

If validation of the release notes can be moved into the sanity check test runs, and they are told that way that their release note is incorrectly structured, that totally works for me.  "Non-trivial" bash for sure, and fragile across OS's per @qu1queee 's observation

Reply:
lets do that. Opening an issue to track that piece of work, thanks folks!